dlhub_sdk/client.py

```diff
@@ -128,39 +128,40 @@ def get_task_status(self, task_id):
         r = self.get("{task_id}/status".format(task_id=task_id))
         return r.json()
 
-    def describe_servable(self, owner, name):
+    def describe_servable(self, name):
```

Comment:
Given these changes are changes in the API, have you updated the Sphinx documentation and the unit tests as well?